src/main.ts

```diff
@@ -1,44 +1,38 @@
-import Vue from 'vue';
-import VueI18n from 'vue-i18n';
-import i18n from './i18n';
+import { App, createApp } from 'vue';
+import { i18n } from './locales';
 import PsAccounts from '@/components/PsAccounts.vue';
 import AccountPanel from '@/components/panel/AccountPanel.vue';
-import {isOnboardingCompleted} from '@/lib/Helpers';
+import { isOnboardingCompleted } from '@/lib/Helpers';
 
-const version = process.env.VUE_APP_VERSION;
+const version = '5.0.0';
```

Comment:
We can't get version from package.json (`process.env.npm_package_version` or by require the package.json file)